### ISSUE TITLE
Conflicting Packages everywhere

### ISSUE THREAD

**Issue by Kelsonk** [airsdk/apm#213]
I've been trying to update a game I haven't touched in awhile. Trying to update all the apms with a ton of issues, so I just created a new project entirely.

I installed the Application and Dialog with no issues, but when trying to install Firebase, I get these conflicts.

CONFLICT :: fatal error : found [3] conflicting packages
CONFLICT :: androidx.core
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Firebase@10.0.0
CONFLICT :: └── 1.10.101 required by: com.distriqt.Application@7.3.1
CONFLICT :: com.distriqt.playservices.Base
CONFLICT :: ├── 19.0.0 required by: com.distriqt.Firebase@10.0.0
CONFLICT :: └── 18.2.4 required by: com.distriqt.Application@7.3.1
CONFLICT :: com.jetbrains.kotlin
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Firebase@10.0.0
CONFLICT :: └── 1.8.220 required by: androidx.core@1.10.101

Don't know what else to try.

**Comment by marchbold** [comment]
We going through a major update of all our extensions so some of the most recent releases will be incompatible. 

You'll need to install the previous version of Firebase currently to be compatible with the extensions that haven't been updated yet, 

To do this:

```
apm view com.distriqt.Firebase
```

And find the previous major release, currently 9.0.4, and install that by:

```
apm install com.distriqt.Firebase 9.0.4
```

**Comment by yvant** [comment]
I've got a similar issue:

CONFLICT :: fatal error : found [5] conflicting packages
CONFLICT :: androidx.appcompat
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.6.104 required by: com.distriqt.facebook.Core@17.0.0
CONFLICT :: androidx.browser
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.5.4 required by: com.distriqt.facebook.Core@17.0.0
CONFLICT :: androidx.core
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.13.100 required by: com.distriqt.facebook.Core@17.0.0
CONFLICT :: androidx.vectordrawable
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.1.6 required by: com.distriqt.facebook.Core@17.0.0
CONFLICT :: com.jetbrains.kotlin
CONFLICT :: ├── 2.0.0 required by: com.distriqt.admob.IronSource@9.0.0
CONFLICT :: └── 1.8.220 required by: com.distriqt.facebook.Core@17.0.0

Does it mean I need to wait for the next facebook ANE update?

**Comment by marchbold** [comment]
Facebook has not been updated yet, you'll need to wait for that update before being able to update the other extensions.

**Comment by Ender22** [comment]
Hey guys, was about to try to update all of my ANE's after this announcement: https://github.com/airsdk/Adobe-Runtime-Support/discussions/3459

But it seems like I can't yet. 

CONFLICT :: fatal error : found [1] conflicting packages
CONFLICT :: androidx.core
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Application@8.0.0
CONFLICT :: └── 1.6.2 required by: com.distriqt.Scanner@5.2.0

Are there plans to update the Scanner ANE soon?

**Comment by Shaunmax** [comment]
I also have similar issue : 

```
CONFLICT :: fatal error : found [6] conflicting packages
CONFLICT :: androidx.appcompat
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.6.104 required by: io.branch.nativeExtensions.Branch@5.4.0
CONFLICT :: androidx.browser
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.5.4 required by: io.branch.nativeExtensions.Branch@5.4.0
CONFLICT :: androidx.core
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.13.100 required by: io.branch.nativeExtensions.Branch@5.4.0
CONFLICT :: com.distriqt.playservices.AdsIdentifier
CONFLICT :: ├── 19.0.0 required by: com.distriqt.Firebase@10.1.0
CONFLICT :: └── 18.1.0 required by: io.branch.nativeExtensions.Branch@5.4.0
CONFLICT :: com.distriqt.playservices.Base
CONFLICT :: ├── 19.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 18.5.0 required by: io.branch.nativeExtensions.Branch@5.4.0
CONFLICT :: com.jetbrains.kotlin
CONFLICT :: ├── 2.0.0 required by: com.distriqt.Adverts@16.0.0
CONFLICT :: └── 1.8.220 required by: androidx.core@1.13.100
```

**Comment by marchbold** [comment]
@Shaunmax I haven't gotten to the Branch ANE yet either. It's on my list.

**Comment by Shaunmax** [comment]
```
CONFLICT :: fatal error : found [2] conflicting packages
CONFLICT :: androidx.core
CONFLICT :: ├── 2.0.1 required by: com.distriqt.Adverts@16.3.0
CONFLICT :: └── 1.6.2 required by: com.distriqt.GoogleAnalytics@4.2.0
CONFLICT :: com.distriqt.playservices.Base
CONFLICT :: ├── 19.1.2 required by: com.distriqt.Adverts@16.3.0
CONFLICT :: ├── 19.0.0 required by: com.distriqt.Application@8.1.0
CONFLICT :: └── 18.0.101 required by: com.distriqt.GoogleAnalytics@4.2.0
```

what could be wrong now ? I am updating for an app after a long gap.

**Comment by marchbold** [comment]
You should remove com.distriqt.GoogleAnalytics, analytics functionality from Google has been replaced by Firebase.

**Comment by Shaunmax** [comment]
> com.distriqt.GoogleAnalytics

you mean uninstall?

**Comment by marchbold** [comment]
Yeah, GA functionality in apps has been moved completely to Firebase
